fesvr/htif.cc

```diff
@@ -105,19 +105,27 @@ static void bad_address(const std::string& situation, reg_t addr)
 std::map<std::string, uint64_t> htif_t::load_payload(const std::string& payload, reg_t* entry)
 {
   std::string path;
+  std::string test_path;
   if (access(payload.c_str(), F_OK) == 0)
     path = payload;
   else if (payload.find('/') == std::string::npos)
   {
-    std::string test_path = PREFIX TARGET_DIR + payload;
+    test_path = PREFIX TARGET_DIR + payload;
     if (access(test_path.c_str(), F_OK) == 0)
       path = test_path;
   }
 
-  if (path.empty())
+  if (path.empty()){
+    if (payload.find('/') == std::string::npos)
+      throw std::runtime_error(
+        "could not open " + test_path +
+        " which is the absolute path spike generated by /$prefix/$with-target/bin/<target program> " + 
```

Comment:
I think it would be easier to understand if you wrote “.  The search path is the current directory, followed by the ‘bin’ directory within the directory specified by the  configure —prefix.”